tribe-ext-show-cost-field.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Plugin Name:       Event Tickets/Eventbrite Extension: Show Cost Field
+ * Description:       Force displaying The Events Calendar's "Event Cost" field in wp-admin and the Community Events event edit form (if applicable) when Event Tickets or Eventbrite Tickets is activated.
+ * Version:           1.0.1
+ * Extension Class:   Tribe__Extension__Show_Cost_Field
+ * GitHub Plugin URI: https://github.com/mt-support/tribe-ext-show-cost-field
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * License:           GPL version 3 or any later version
+ * License URI:       https://www.gnu.org/licenses/gpl-3.0.html
+ * Text Domain:       tribe-ext-show-cost-field
+ *
+ *     This plugin is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     any later version.
+ *
+ *     This plugin is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *     GNU General Public License for more details.
+ */
+
+// Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+if (
+	class_exists( 'Tribe__Extension' )
+	&& ! class_exists( 'Tribe__Extension__Show_Cost_Field' )
+) {
+	/**
+	 * Extension main class, class begins loading on init() function.
+	 */
+	class Tribe__Extension__Show_Cost_Field extends Tribe__Extension {
+
+		/**
+		 * Setup the Extension's properties.
+		 *
+		 * This always executes even if the required plugins are not present.
+		 */
+		public function construct() {
+			$this->set_url( 'https://theeventscalendar.com/extensions/show-cost-field/' );
```

Comment:
How about removing this and adding a `Plugin URI` field to the header?

Reply:
done